parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -128,7 +128,10 @@ fi
 
 setupCNIDirs
 
-logs_to_events "AKS.CSE.installNetworkPlugin" installNetworkPlugin
+# Network plugin should not install if outbound is not allowed.
+if [ "${BLOCK_OUTBOUND_NETWORK}" == "false" ]; then
+    logs_to_events "AKS.CSE.installNetworkPlugin" installNetworkPlugin
```

Comment:
have we re-done things to make sure we still get a valid CNI plugin even if RP wants a different version than what's cached, and we want to block outbound network?

Reply:
I'm going to take this portion out of the PR for now, since I would like to get the BLOCK_OUTBOUND_NETWORK var in. But I think I made a mistake here, I'll need to double check, but we should just be getting an alternative URL to download CNI